db/database_stats.go

```diff
@@ -21,7 +21,7 @@ package db
 
 // Update database-specific stats that are more efficiently calculated at stats collection time
 func (db *DatabaseContext) UpdateCalculatedStats() {
-	if !db.onlyDefaultCollection() {
+	if !db.OnlyDefaultCollection() {
```

Comment:
Is this needed?  Seems like hasDefaultCollection may be sufficient.

Reply:
legacy code, i only changed this because I wanted to make OnlyDefaultCollection public for rest.